### PR TITLE
Update beta phase timeline

### PR DESCRIPTION
**Pull request by zkamvar**
this reflects the updated schedule of the workbench beta phase as outlined in https://carpentries.org/blog/2022/12/dovetail-12/

Additionally, please let me know if you would like me to reset your test repository at https://github.com/fishtree-attempt/R-ecology-lesson.

Best,
Zhian

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203516117412888

### CODE REVIEW

**Comment by zkamvar** [comment]
Hello @datacarpentry/r-ecology-lesson-maintainers,

I wanted to add a friendly reminder about this PR that should be merged.

File: before_body.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <a href='https://carpentries.github.io/workbench'>The Carpentries Workbench</a>:
   <a href="https://preview.carpentries.org/R-ecology-lesson">https://preview.carpentries.org/R-ecology-lesson</a>.
   <br>
-  <b>The Workbench version of this lesson will become default on 2023-01-23</b>.
+  <b>The Workbench version of this lesson will become default on 2023-04-03</b>.
```
Comment (ErinBecker) [inline before_body.html:6]:
```suggestion
  <b>The Workbench version of this lesson will become default on 2023-05-01</b>.
```

**Comment by ErinBecker** [comment]
Thanks @zkamvar for this PR. I've updated the date to reflect what is now shown in the [transition schedule](https://carpentries.github.io/workbench/transition-schedule.html).

**Comment by zkamvar** [comment]
Thank you Erin!

**Pull request merged**